src/main/java/at/helpch/placeholderapi/expansion/player/util/ItemUtil.java

```diff
@@ -0,0 +1,97 @@
+package at.helpch.placeholderapi.expansion.player.util;
+
+import org.bukkit.Material;
+import org.bukkit.NamespacedKey;
+import org.bukkit.enchantments.Enchantment;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.inventory.meta.Damageable;
+import org.bukkit.inventory.meta.ItemMeta;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Optional;
+
+@SuppressWarnings("deprecation")
+public final class ItemUtil {
+
+    /**
+     * Whether an item is null or its type is air (ends with _AIR)
+     *
+     * @param itemStack item to check
+     * @return whether the item is null or air
+     */
+    public static boolean isNullOrAir(@Nullable final ItemStack itemStack) {
+        return itemStack == null || itemStack.getType().name().contains("_AIR");
```

Comment:
Please note that `AIR` is also a valid material. Shouldn't it be checked against?

Reply:
Yeah.. I've missed that 😬